qa/suites/fs/volumes/tasks/volumes/test/emptystring.yaml

```diff
@@ -0,0 +1,5 @@
+tasks:
+  - cephfs_test_runner:
+      fail_on_skip: false
+      modules:
+        - tasks.cephfs.test_volumes.TestEmptyStringForCreates
```

Comment:
IMO it's better to keep tests under `TestEmptyStringForCreate` as a part of "basic" group `fs:volumes` suite. IOW, adding `TestEmptyStringForCreate` to  `qa/suites/fs/volumes/tasks/volumes/test/basic.yaml` is a better idea

Reply:
moved YAML snippet to qa/suites/fs/volumes/tasks/volumes/test/basic.yaml